app/views/forms/show.html.erb

```diff
@@ -30,23 +28,17 @@
       </div>
     </dl>
 
-    <% if @pages.any? %>
-      <h2 class="govuk-heading-m govuk-!-margin-top-7 govuk-!-margin-bottom-0"><%= t("forms.form_overview.your_questions") %></h2>
-      <% @pages.each do |page| %>
-        <dl class="govuk-summary-list">
-          <div class="govuk-summary-list__row">
-            <dt class="govuk-summary-list__key govuk-!-font-weight-regular">
-              <%= page.question_text %>
-            </dt>
-            <dd class="govuk-summary-list__actions">
-              <%= link_to edit_page_path(@form,page), class: "govuk-link govuk-link--no-visited-state" do %>
-              <%= t("forms.form_overview.edit") %> <span class="govuk-visually-hidden"><%= page.question_text %></span>
-              <% end %>
-            </dd>
-          </div>
-        </dl>
-      <% end %>
-    <% end %>
+    <h2 class="govuk-heading-m govuk-!-margin-top-7 govuk-!-margin-bottom-0"><%= t("forms.form_overview.your_questions") %></h2>
+
+    <dl class="govuk-summary-list">
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key govuk-!-font-weight-regular">
+          <%= govuk_link_to t("forms.form_overview.add_or_edit_questions"), form_pages_path(@form) %>
```

Comment:
So app/views/forms/show.html.erb is going to become the task list right?

(Not a blocking comment, just making sure I understand properly!)

Reply:
yeah that is correct